app/src/lib/stores/repository-settings-store.ts

```diff
@@ -110,6 +110,12 @@ async function formatGitIgnoreContents(
       return
     }
 
+    // fallback to Git default behaviour
```

Comment:
I think this should be a part of an else block since it's very much related to the previous condition.